<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1301</width>
      <height>801</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Acceuil&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Acceuil</string>
     </attribute>
     <widget class="QWidget" name="widget_15" native="true">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>-30</y>
        <width>1301</width>
        <height>831</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(251, 251, 243);
font: 700 9pt &quot;Segoe UI&quot;;
border: 1px solid #000; 
border-color: rgb(251, 251, 243);

</string>
      </property>
      <widget class="QWidget" name="widget_16" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>841</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(212, 153, 162);
    border-radius: 50px;
    border: 1px solid #000; 
    padding: 5px;
</string>
       </property>
       <widget class="QLabel" name="label_82">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>id transaction:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_83">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Date d'ajout &amp; heure :</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="date">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>70</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_84">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>140</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Montant article:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_85">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Listes des articles:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="box">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>260</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Virement Bancaire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Espèces</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chèque</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Carte de Crédit/Débit</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 168, 213);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="annuler">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>300</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 168, 213);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_86">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>250</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(212, 153, 162);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../Downloads/Capture_d_écran_2024-10-06_133623-removebg-preview.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_87">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>250</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(211, 152, 161);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Methode de payement:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_88">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Taxes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_89">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>30</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Reduction:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="l1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="l6">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>70</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="l3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>180</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="l4">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>70</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="l2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>180</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_90">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>CIN:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="l7_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="l3_3">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>140</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Status :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="l8">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>180</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>300</y>
          <width>31</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tri">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>300</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_92">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 131, 143);
font: 700 10pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Ajout de transaction:</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_18" native="true">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>590</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
                                                    color: rgb(1, 4, 8);
                                                    background-color: #fdefd5;
                                                    font-family: Arial, Helvetica, sans-serif;
                                                    font-size: 16px;
                                                    line-height: 1.5;
                                                    width: 100px;
                                                    height: 40px;
                                                    opacity: 0.65;
                                                    text-align: center;
                                                    border-color: rgb(1, 4, 8);
                                                    border-width: 3px;
                                                    border-style: solid;
                                                    border-width: 0.125em;
                                                    border-radius: 0.5em;
                                                    border-bottom-width: 4px;
                                                    border-right-width: 4px;
                                                    }
                                                    QLineEdit:hover{
                                                    color: rgb(1, 4, 8);
                                                    background-color: #f5f2f2;
                                                    border-color: rgb(24, 49, 83);
                                                    border-bottom-width: 3px;
                                                    border-right-width: 3px;
                                                    }
                                                    QLineEdit:pressed{
                                                    background-color: #cccccc;
                                                    border-color: rgb(24, 49, 83);
                                                    }</string>
       </property>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 168, 213);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_20" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>430</y>
         <width>401</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(209, 168, 213);
    border-radius: 50px;
    border: 1px solid #000; 
    padding: 5px;

</string>
       </property>
       <widget class="QLabel" name="label_95">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(211, 152, 161);
font: 700 8pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>ID recherhcer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherche">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>400</y>
         <width>741</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(212, 153, 162);

    padding: 5px;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>630</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(209, 168, 213);
font: 700 8pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l7">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>630</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(212, 153, 162);
    border-radius: 50px;
    border: 1px solid #000; 
    padding: 5px;
</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>100</y>
         <width>331</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="transactionLayout"/>
      </widget>
      <widget class="QPushButton" name="statistiqueButtonTransaction">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>60</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 131, 143);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(209, 168, 213);
    border-radius: 50px;
    border: 1px solid #000; 
    padding: 5px;</string>
       </property>
       <property name="text">
        <string>statistique par status</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>vocale</string>
     </attribute>
     <widget class="QWidget" name="widget_23" native="true">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>1301</width>
        <height>791</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(251, 251, 243);
border: 1px solid #000; 
border-color: rgb(251, 251, 243);

</string>
      </property>
      <widget class="QPushButton" name="vocale">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 131, 143);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(209, 168, 213);</string>
       </property>
       <property name="text">
        <string>vocale</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_voc">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>90</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(209, 168, 213);
color: rgb(0, 131, 143);
font: 700 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="voc_text">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>180</y>
         <width>301</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(212, 153, 162);
    border-radius: 50px;
    border: 1px solid #000; 
    padding: 5px;
</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="statistique_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>730</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 131, 143);
font: 700 11pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>statistique par status</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
